### DISCUSSION TITLE
Quick action to copy post link

### DISCUSSION THREAD

**Discussion by beyond9thousand** [Docile-Alligator/Infinity-For-Reddit#1305] [Ideas]
[Megalodon](https://github.com/sk22/megalodon) has this feature (nothing revolutionary) where you can long press the share button to copy the post link to your clipboard. Could this be implemented in infinity?

**Comment by camporter** [comment]
I liked this so created https://github.com/Docile-Alligator/Infinity-For-Reddit/pull/1313

**Reply by Wladefant** [reply]
Thank you for your enthusiasm, but maybe you could ask first. Because not everything is in our scope and it would be sad to close your pull request. But if that doesn't bother you, feel free to create pull requests directly haha

**Reply by camporter** [reply]
I added the PR since I include it in my personal build since I found it useful. I don't expect anything to be merged or used, just want to provide the change for others who might want it and maybe encourage some discussion on the PR. I'm always open to suggestions, changes, or further discussion!

**Reply by Wladefant** [reply]
Haha all right,  understood. Happy to have you!!!

**Comment by beyond9thousand** [comment]
@camporter sweet!
